Sources/ApolloCodegenLib/FileGenerators/OperationManifestFileGenerator.swift

```diff
@@ -14,7 +14,7 @@ struct OperationManifestItem {
 
     var source = operation.definition.source.convertedToSingleLine()
     for fragment in operation.referencedFragments {
-      source += "\\n\(fragment.definition.source.convertedToSingleLine())"
+      source += #"\\n\#(fragment.definition.source.convertedToSingleLine())"#
```

Comment:
This looks odd to me, but if the file comes out formatted correctly then that's fine. It seems like you have unbalanced `#` and I thought string interpolation wouldn't work inside of a raw string.

Reply:
This fix appears to have broken communication with the back end. Hash generated incorrectly